### DISCUSSION TITLE
Can you add the contents dropdown to the sidebar?

### DISCUSSION THREAD

**Discussion by JackAshwell11** [pradyunsg/furo#294] [Questions] [answered]
So I have an .rst file which contains the my API reference and when viewing that reference, it shows a contents dropdown in the top right corner. So can I have this dropdown in the sidebar as well as the top right corner? Or maybe have the contents dropdown stay in the top right as I'm scrolling.

**Comment by JackAshwell11** [comment, marked as answer]
Never mind, I found out how. You need the toctree to link to another rst file which has a toctree linking to multiple files which each hold the documentation.

**Reply by pradyunsg** [reply]
Yup!
